script/service/tasklist-back.js

```diff
@@ -63,7 +61,15 @@ angular.module('tasklist-back', ['utils']).
                 return ret;
             };
 
```

Comment:
I think you should split methods that belong to task (changeTaskStatus, changeTaskDescription, etc.) and methods that belong to tasklist(add, remove, find). To make you API more clear and structured.